src/psyclone/tests/domain/lfric/lfric_builtins_test.py

```diff
@@ -2881,6 +2892,26 @@ def test_int_X(tmpdir, monkeypatch, annexed, dist_mem):
         assert output_dm_2 in code
 
 
+def test_int_X_precision(monkeypatch):
+    '''Test that the builtin picks up and creates correct code for a
+    scalar with precision that is not the default i.e. not i_def. At
+    the moment there is no other integer precision so we make one
+    up. However, this does mean that we can't check whether it
+    compiles.
```

Comment:
I would have expected PSyclone to reject an unrecognised precision. Are there rules about precision in the LFRic API - i.e. is there a specified list of precision parameters that must be used?

Reply:
The value is monkeypatched in to avoid any tests. I've tried to make that clear in the text.